src/shmem_internal.h

```diff
@@ -337,7 +337,7 @@ void* shmem_internal_get_next(intptr_t incr);
 static inline double shmem_internal_wtime(void) {
     double wtime = 0.0;
 
-#ifdef CLOCK_MONOTONIC
+#ifdef HAVE_CLOCK_GETTIME
```

Comment:
don't we additionally need to check if CLOCK_MONOTONIC is also defined? or was that picked up by your configure.ac check? Also we run into this same issue in the perf tests